movie_api/cinema/migrations/0002_rename_cinema_movie.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 5.0.2 on 2024-02-19 14:50
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("cinema", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.RenameModel(
+            old_name="Cinema",
+            new_name="Movie",
+        ),
+    ]
```

Comment:
It is a good practice to include one migration per app per PR to not overload `migrations` folder. Please, add one migration to the folder.